evalml/tests/utils_tests/test_dependencies.py

```diff
@@ -9,7 +9,7 @@ def test_has_minimal_deps(has_minimal_dependencies):
     lines = open(reqs_path, 'r').readlines()
     lines = [line for line in lines if '-r ' not in line]
     reqs = requirements.parse(''.join(lines))
-    extra_deps = [req.name for req in reqs]
+    extra_deps = [req.name if req.name != 'imbalanced-learn' else 'imblearn' for req in reqs]
```

Comment:
The pip install is for `imbalanced-learn` but the import itself is `imblearn`, which is why I added this conditional here

Reply:
Why is this not an issue with scikit-learn? Same patten, scikit-learn is the pip install but import is sklearn 🤔

Reply:
I think it's because this test looks only at the extra requirements in the `requirements.txt` file, not including the modules in `core_requirements.txt`, which is where scikit-learn is.

Reply:
👍 

Once we add full support for oversampling, I hope we can either move `imbalanced-learn` to `core-requirements.txt` or remove it in favor of our own impls. At that point, this won't be an issue, right?